base/multi.jl

```diff
@@ -1060,7 +1060,18 @@ function process_tcp_streams(r_stream::TCPSocket, w_stream::TCPSocket, incoming)
     message_handler_loop(r_stream, w_stream, incoming)
 end
 
-function process_messages(r_stream::IO, w_stream::IO, incoming=true)
+"""
+    Base.process_messages(r_stream::IO, w_stream::IO, incoming::Bool=true)
+
+Called by cluster managers using custom transports. It should be called when the custom
+transport implementation receives the first message from a remote worker. The custom
+transport must manage a logical connection to the remote worker and provide two
+`IO` objects, one for incoming messages and the other for messages addressed to the
+remote worker.
+If `incoming` is `true`, schedules reads from `r_stream` and writes to `w_stream`.
+Otherwise, schedules in the reverse direction.
```

Comment:
Thanks for explaining! This is why I wanted to update the docs - it wasn't clear to me what was going on.